.github/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Release 0.2.5 (development release)
+
+<h3>New features since last release</h3>
+
+<h3>Improvements</h3>
+
+<h3>Breaking changes</h3>
+
+<h3>Bug fixes</h3>
+
+* Templates are no longer changed when calling them and replacing the parameters with actual values.
+
+  ```python
+  bb_program = bb_template(**template_variables)
+  ```
+
+<h3>Documentation</h3>
+
```

Comment:
```suggestion
<h3>Documentation</h3>
```

Reply:
What's the change here? It looks the same. :thinking:

Reply:
Oh, it's removing the double new line after 'Documentation'; GitHub doesn't render it very well